packages/contracts/contracts/plugin/PluginSetupProcessor.sol

```diff
@@ -661,19 +650,22 @@ contract PluginSetupProcessor is DaoAuthorizable {
             DAO(payable(_dao)).applyMultiTargetPermissions(_params.permissions);
         }
 
-        emit UninstallationApplied({dao: _dao, plugin: _params.plugin, setupId: setupId});
+        emit UninstallationApplied({dao: _dao, plugin: _params.plugin, setupId: preparedSetupId});
```

Comment:
Are we sure that we will never need
```solidity
        bytes32 appliedSetupId = _getAppliedSetupId(
            _params.pluginSetupRef,
            ZERO_BYTES_HASH
        );
```
to be emitted with `event UninstallationApplied`?

Reply:
I don't think we need it, but in case we might need it, it can still be figured out ! by `preparedSetupId`, you can get the version and thats it.